File: src/main/java/com/asmsi/admin_system/entity/Student.java
package com.asmsi.admin_system.entity;

import java.time.LocalDate;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Data
public class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String lrn;
    private String idNumber;
    private String name;
    private LocalDate birthday;
    private String address;
    private String placeOfBirth;
    private String elementarySchool;
    private String via;
    private String campus;
    private String familySaint;
    private String batchNumber;
    
    // New Field for Image
    private String imageUrl;
}

File: src/main/resources/static/js/student.js
module.exports = (sequelize, DataTypes) => {
    return sequelize.define('Student', {
      address: DataTypes.STRING,
      birthday: DataTypes.STRING,
      campus: DataTypes.STRING,
      elementary_school: DataTypes.STRING,
      id_number: DataTypes.STRING,
      lrn: DataTypes.STRING,
      name: DataTypes.STRING,
      place_of_birth: DataTypes.STRING,
      via: DataTypes.STRING,
      family_saint: DataTypes.STRING,
      batch_number: DataTypes.STRING,
      image_url: DataTypes.STRING,
    });
  };
  